### PR TITLE
Responsive error of About Page 

### LINKED ISSUE #1656: Responsive Error of About Page
### What happened just before the problem occurred
The About Page of Mapknitter is not responsive according to screen resolution

https://user-images.githubusercontent.com/67657066/163667678-a886f8fd-0e9c-44e8-bb47-fd8249764680.mp4

### Relevant URLs

https://https://mapknitter.org/about

### PR DESCRIPTION
**Pull request by asaayushisingh**
Fixes #1656  (<=== Add issue number here)
**Problem**
The About Page of Mapknitter is not responsive according to screen resolution

### Relevant URLs

https://https://mapknitter.org/about

**Before**

https://user-images.githubusercontent.com/67657066/163668837-ed91ebe9-8cb5-4222-98fe-aaad25edddf6.mp4

**After**-->

https://user-images.githubusercontent.com/67657066/163668852-4f799137-115d-4ddf-93f5-01f1e3183207.mp4

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

### CODE REVIEW

**Comment by asaayushisingh** [comment]
Hey! @cesswairimu, Kindly review this! Thank You!

File: app/assets/stylesheets/header.scss [hunk 1]
```diff
@@ -41,6 +41,10 @@ body {
 }
 
 @media(max-width: 992px) {
+  #about-img {
+    margin: 20px auto;
+    width: -webkit-fill-available;
```
Comment (on2onyekachi) [inline app/assets/stylesheets/header.scss:46]: Hi @asaayushisingh nice approach! though using 'width: -webkit-fill-available' will be experienced only for chrome users, leaving out firefox, IE, and other users. good use will be to set.
 ```
    min-width: 100%;
     min-width: -moz-available;          
     min-width: -webkit-fill-available;  
     min-width: fill-available;
```

Also, Try to check this on other browsers before you make changes.

**Review by on2onyekachi — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/header.scss (hunk 1 above)
Comment (on2onyekachi) [reply app/assets/stylesheets/header.scss:46]: Also ```width: 100%``` should be a good fix.

Comment (asaayushisingh) [reply app/assets/stylesheets/header.scss:46]: Hey! @on2onyekachi  Thanks a lot for pointing this out.

Comment (asaayushisingh) [reply app/assets/stylesheets/header.scss]: Hey! @on2onyekachi I had made the required changes. Kindly review it, Thank You!

**Comment by on2onyekachi** [comment]
@asaayushisingh Yeah, great!.

**Comment by asaayushisingh** [comment]
Hey! @cesswairimu, Kindly review this! Thank You!

**Comment by asaayushisingh** [comment]
Hey! @cesswairimu, Kindly review this! Thank You!

**Review by cesswairimu — APPROVED** [review]
Great :tada: Thanks for raising and fixing this @asaayushisingh :heart: :heart:

**Pull request merged**